apps/theming-designer/src/components/FabricSlotWidget.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { IThemeRules, FabricSlots, IThemeSlotRule } from 'office-ui-fabric-react/lib/ThemeGenerator';
+import { IColor } from 'office-ui-fabric-react/lib/Color';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { mergeStyles } from '@uifabric/merge-styles';
+import { ColorPicker } from 'office-ui-fabric-react/lib/ColorPicker';
+import { Callout, DirectionalHint } from 'office-ui-fabric-react/lib/Callout';
+import { Text } from 'office-ui-fabric-react';
+
+export interface IFabricSlotWidgetProps {
+  slot: FabricSlots;
+  onFabricPaletteColorChange: (newColor: IColor | undefined, fabricSlot: FabricSlots) => void;
```

Comment:
Is `newColor` ever `undefined`? It doesn't seem to be when looking at where it is invoked:

https://github.com/OfficeDev/office-ui-fabric-react/pull/9734/files#diff-32b8c034787398017ffdcc2246a720c8R84

If not, I recommend removing `undefined` from the typing.

Reply:
done